articles/150_ros_command_line_arguments.md

```diff
@@ -167,16 +167,24 @@ another_node:
      string_param: bar
 ```
 
-#### Logging level
+#### Multiple Logging level assignments
 
-Minimum logging level can be externally set using the `--log-level` option.
+Logging level assignments have two means of settings which are a global logging level and other logging levels for specific loggers. Minimum logging level can be externally set using the `--log-level` option.
```

Comment:
`a global logging level` means `a global minimal logging level`, right?. I think either global or not, `Minimum` is intended to described by author.

Reply:
Yes, the logging level in ROS2 uses a minimum value to control the output message.
`minimum` is used for both the global logging level and the specific logger level.

Reply:
nit: I suggest rephrasing, as it sounds a bit confusing to me.
```suggestion
Minimum logging level can be externally set either globally or per logger using the `--log-level` option.
```